### PR TITLE
Update to node16

### PR DESCRIPTION
**Pull request by mattgauntseo-sentry**

### CODE REVIEW

**Review by joshuarli — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "format-check": "yarn prettier --check **/*.ts",
     "lint": "eslint src/**/*.ts",
     "dist": "yarn build && yarn ncc build",
-    "test": "yarn jest",
+    "test": "yarn jest --passWithNoTests",
```
Comment (shenotme) [inline package.json:12]: Node

File: package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
...
     "all": "yarn build && yarn format && yarn lint && yarn test && yarn dist"
```
Comment (shenotme) [inline package.json:13]: Bode

File: action.yml [hunk 3]
```diff
@@ -16,5 +16,5 @@ outputs:
   token:
     description: 'Github Token for App installation'
 runs:
```
Comment (shenotme) [inline action.yml:18]: End

File: action.yml [hunk 4]
```diff
@@ -16,5 +16,5 @@ outputs:
...
-  using: 'node12'
```
Comment (shenotme) [inline action.yml:19]: End

File: action.yml [hunk 5]
```diff
@@ -16,5 +16,5 @@ outputs:
...
+  using: 'node16'
```
Comment (shenotme) [inline action.yml:19]: End

File: action.yml [hunk 6]
```diff
@@ -16,5 +16,5 @@ outputs:
...
   main: 'dist/index.js'
```
Comment (shenotme) [inline action.yml:20]: End

File: package.json (hunk 2 above)
Comment (shenotme) [inline package.json:13]: End

File: package.json [hunk 7]
```diff
@@ -9,7 +9,7 @@
...
   },
   "repository": {
```
Comment (shenotme) [inline package.json:15]: End

File: package.json [hunk 8]
```diff
@@ -45,15 +45,15 @@
     "node-fetch": "^2.6.1"
   },
   "dependencies": {
-    "@actions/core": "^1.5.0",
-    "@octokit/auth-app": "^3.6.0",
-    "@octokit/rest": "^18.9.1"
```
Comment (shenotme) [inline package.json:50]: Ending

File: package.json [hunk 9]
```diff
@@ -45,15 +45,15 @@
     "node-fetch": "^2.6.1"
```
Comment (shenotme) [inline package.json:45]: End

Comment (shenotme) [inline package.json:45]: End

File: package.json [hunk 10]
```diff
@@ -45,15 +45,15 @@
...
-    "@octokit/rest": "^18.9.1"
+    "@actions/core": "^1.10.0",
+    "@octokit/auth-app": "^4.0.7",
+    "@octokit/rest": "^19.0.5"
   },
   "devDependencies": {
     "@types/jest": "^27.0.1",
     "@types/node": "^12.7.12",
     "@typescript-eslint/parser": "^4.30.0",
-    "@vercel/ncc": "^0.30.0",
```
Comment (shenotme) [inline package.json:56]: End

File: package.json [hunk 11]
```diff
@@ -45,15 +45,15 @@
...
+    "@vercel/ncc": "^0.34.0",
     "eslint": "^7.32.0",
     "eslint-config-sentry": "^1.60.0",
     "eslint-plugin-github": "^4.2.0",
```
Comment (shenotme) [inline package.json:59]: End

File: package.json [hunk 12]
```diff
@@ -45,15 +45,15 @@
...
+    "@octokit/rest": "^19.0.5"
   },
   "devDependencies": {
     "@types/jest": "^27.0.1",
     "@types/node": "^12.7.12",
     "@typescript-eslint/parser": "^4.30.0",
-    "@vercel/ncc": "^0.30.0",
+    "@vercel/ncc": "^0.34.0",
     "eslint": "^7.32.0",
     "eslint-config-sentry": "^1.60.0",
```
Comment (shenotme) [inline package.json:58]: End

File: package.json (hunk 11 above)
Comment (shenotme) [reply package.json:59]: End

File: package.json [hunk 13]
```diff
@@ -9,7 +9,7 @@
     "format-check": "yarn prettier --check **/*.ts",
     "lint": "eslint src/**/*.ts",
     "dist": "yarn build && yarn ncc build",
-    "test": "yarn jest",
```
Comment (shenotme) [inline package.json:12]: End

File: package.json [hunk 14]
```diff
@@ -45,15 +45,15 @@
     "node-fetch": "^2.6.1"
   },
   "dependencies": {
-    "@actions/core": "^1.5.0",
-    "@octokit/auth-app": "^3.6.0",
```
Comment (shenotme) [inline package.json:49]: End

File: package.json [hunk 15]
```diff
@@ -45,15 +45,15 @@
     "node-fetch": "^2.6.1"
   },
   "dependencies": {
-    "@actions/core": "^1.5.0",
```
Comment (shenotme) [inline package.json:48]: End

Comment (shenotme) [inline package.json:48]: End

Comment (shenotme) [inline package.json:48]: End

**Pull request merged**